admin-dev/themes/default/sass/partials/_toolbar.sass

```diff
@@ -10,7 +10,10 @@
 	color: #363A41
 	padding: 0 0 0 210px
 
-	@media (max-width: 1024px)
+	.mobile &
+		top: $size-header-height + ($header-mobile-padding-y * 2)
```

Comment:
Can you remove tabs and add spaces instead? (In an other commit ofc)

Reply:
I can, but not in this PR 😄 
I'd like to upgrade to webpack 4, too

Reply:
:+1: